### PR TITLE
fix(certificate): correcting format and removing unnecessary config

### LINKED ISSUE #783: Fix certificate format and remove unnecessary fields
We are seeing Kong gets out of sync in ArgoCD due to some wrong formating and 
unnecessary fields

https://github.com/Kong/charts/blob/7d1047fcc4ab33895821beb9d5d66e5f22639369/charts/kong/templates/certificate.yaml#L59-L61

These lines will be automatically formatted and changed to

```
  renewBefore: 360h0m0s
  duration: 2160hh0m0s
```
and also removing is `isCa: false`

Check ArgoCD validation diff for clarity, 
image

Is this possible to fix this to not have kong get out of sync in ArgoCD?

### PR DESCRIPTION
**Pull request by philipsabri**
#### What this PR does / why we need it:
Fix correct timestamp format
Remove isCA as it is not necessary to configure it as false

#### Which issue this PR fixes
  - fixes #783 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] PR is based off the current tip of the `main` branch.
- [ ] Changes are documented under the "Unreleased" header in CHANGELOG.md
- [ ] New or modified sections of values.yaml are documented in the README.md
- [ ] Commits follow the [Kong commit message guidelines](https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#commit-message-format)

### CODE REVIEW

**Comment by pmalek** [comment]
Hi @philipsabri 👋 

Thanks for the patch! I've triggered the CI run but in the meantime so you mind adding a changelog entry and bumping the version https://github.com/Kong/charts/blob/dd8bae5f59be843653ded530cdf2bf988f2ab8c2/charts/kong/Chart.yaml#L14 to 2.20.1 ?

**Comment by philipsabri** [comment]
> Hi @philipsabri 👋
> 
> Thanks for the patch! I've triggered the CI run but in the meantime so you mind adding a changelog entry and bumping the version
> 
> https://github.com/Kong/charts/blob/dd8bae5f59be843653ded530cdf2bf988f2ab8c2/charts/kong/Chart.yaml#L14
> 
> to 2.20.1 ?

Thanks for the quick reply, I've updated the chart.yaml

**Comment by pmalek** [comment]
> > Hi @philipsabri 👋
> > Thanks for the patch! I've triggered the CI run but in the meantime so you mind adding a changelog entry and bumping the version
> > https://github.com/Kong/charts/blob/dd8bae5f59be843653ded530cdf2bf988f2ab8c2/charts/kong/Chart.yaml#L14
> > 
> > to 2.20.1 ?
> 
> Thanks for the quick reply, I've updated the chart.yaml

Let's add the changelog entry as well https://github.com/Kong/charts/blob/main/charts/kong/CHANGELOG.md and I believe we should be good to go. Thanks!

**Comment by philipsabri** [comment]
> > > Hi @philipsabri 👋
> > > Thanks for the patch! I've triggered the CI run but in the meantime so you mind adding a changelog entry and bumping the version
> > > https://github.com/Kong/charts/blob/dd8bae5f59be843653ded530cdf2bf988f2ab8c2/charts/kong/Chart.yaml#L14
> > > 
> > > to 2.20.1 ?
> > 
> > 
> > Thanks for the quick reply, I've updated the chart.yaml
> 
> Let's add the changelog entry as well https://github.com/Kong/charts/blob/main/charts/kong/CHANGELOG.md and I believe we should be good to go. Thanks!

Ah sorry missed that one, updated it now. Please come back with any suggestion if the CHANGELOG format should be changed

File: charts/kong/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2.20.1
+
+### Fixed
+
+* Fix correct timestamp format and remove ``isCA`` in certificates [#791](https://github.com/Kong/charts/pull/791)
```
Comment (pmalek) [inline charts/kong/CHANGELOG.md]:
```suggestion
* Fix correct timestamp format and remove `isCA` in certificates
  [#791](https://github.com/Kong/charts/pull/791)
```

**Review by pmalek — APPROVED** [review]

**Pull request merged**